### DISCUSSION TITLE
Problems using MacOS 12.6

### DISCUSSION THREAD

**Discussion by ftcastro** [curl/curl#9693] [Q&A]
Hello!

I am having problems with curl and github on MacOS 12.6. When I try to clone a project using the PyCharm IDE, I receive the following error message: unable to find remote helper for 'https'

When I try to do it using it via line code, I receive the same message, but sometimes, the following error: github.com: errno=Operation timed out

Can anyone please help me?

Best regards,
Fernando.

**Comment by jay** [comment]
> unable to find remote helper for 'https'

That looks like a git problem.

> github.com: errno=Operation timed out

That looks like a network problem.

I suggest reinstall git and try again.

**Reply by ftcastro** [reply]
Hello Jay, thanks for the reply. It doesn't seem to be a network problem, at least the computer is accessing the internet correctly. My guess is that the git is not communnicating with the curl. I have both installed (checked via Terminal). Is there any way to check my git configurations?

Also, I'm not a very experienced mac user. What command do I type to uninstall git using the Terminal?

Best regards,
Fernando.

**Reply by nickzman** [reply]
On macOS, if you didn't install git yourself, you probably got it from Homebrew or Xcode. If the former, then type `brew reinstall git` into the command line. If the latter, then delete Xcode.app from the Applications directory, then use the Mac App Store to reinstall it.
